metaverse_components/pet.js

```diff
@@ -108,15 +111,14 @@ export default (app, component) => {
           rootBone.updateMatrixWorld();
         }
         if (petMixer) { // animated pet
-          if (petSpec) { // activated pet
+          if (petSpec && player) { // activated pet

```

Comment:
Should remove these checks if `player` cannot be null.

That would mean that the worn app is not part of the player's app manager, which seems it would be a bug.